### PR TITLE
migrations: Add AdoptInstances to lxd provider

### PR DESCRIPTION
**Pull request by babbageclunk**
Part of https://bugs.launchpad.net/juju/+bug/1648063

Implement AdoptInstances in the lxd provider using client.SetContainerConfig.

QA steps - none, this code isn't runnable until the migration master is updated.

### CODE REVIEW

**Comment by babbageclunk** [comment]
!!chittychitty!!

File: provider/lxd/environ_instance.go [hunk 1]
```diff
@@ -123,3 +123,22 @@ func (env *environ) parsePlacement(placement string) (*instPlacement, error) {
 
 	return nil, errors.Errorf("unknown placement directive: %v", placement)
 }
+
+// MoveInstancesToController updates the controller tags on the
```
Comment (axw) [inline provider/lxd/environ_instance.go]: MoveInstancesToController is a bit of a mouthful. How about "AdoptInstances", to keep it brief?

Comment (babbageclunk) [inline provider/lxd/environ_instance.go]: Yeah, I think I prefer AdoptInstances.

**Comment by babbageclunk** [comment]
$$merge$$

**Comment by babbageclunk** [comment]
Seems spurious - it's lxd failing to bootstrap, but I can bootstrap locally fine.
$$merge$$

**Pull request merged**